src/T2DMap.cpp

```diff
@@ -4097,6 +4097,11 @@ void T2DMap::slot_setArea()
     }
     set_room_area_dialog->setAttribute(Qt::WA_DeleteOnClose);
     arealist_combobox = set_room_area_dialog->findChild<QComboBox*>("arealist_combobox");
+    auto label_info = set_room_area_dialog->findChild<QLabel*>("label_info");
+    auto font = QFont(QApplication::font());
```

Comment:
IIRC just declaring something as a `QFont` will use the application font anyhow. I *think* this will do the same:
```suggestion
    QFont font;
```
though I guess it might be worth checking before relying on it... :wink:

Reply:
Yep! This is indeed what doc says and it's working as expected that way.